File: tl/src/main/java/com/github/badoualy/telegram/tl/api/TLAbsWallPaper.java

package com.github.badoualy.telegram.tl.api;


import com.github.badoualy.telegram.tl.core.TLObject;



public abstract class TLAbsWallPaper extends TLObject {

    protected int id;

    protected String title;

    protected int color;


    public int getId() {
        return id;
    }

    public void setId(int value) {
        this.id = value;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String value) {
        this.title = value;
    }

    public int getColor() {
        return color;
    }

    public void setColor(int value) {
        this.color = value;
    }

}

File: tl/src/main/kotlin/com/github/badoualy/telegram/tl/api/TLAbsWallPaper.kt
package com.github.badoualy.telegram.tl.api

import com.github.badoualy.telegram.tl.core.TLObject

/**
 * Abstraction level for the following constructors:
 * * [wallPaper#ccb03657][TLWallPaper]
 * * [wallPaperSolid#63117f24][TLWallPaperSolid]
 *
 * @author Yannick Badoual yann.badoual@gmail.com
 * @see <a href="http://github.com/badoualy/kotlogram">http://github.com/badoualy/kotlogram</a>
 */
abstract class TLAbsWallPaper : TLObject() {
    abstract var id: Int

    abstract var title: String

    abstract var color: Int
}
